packages/dashboard-core-plugins/src/ConsolePlugin.tsx

```diff
@@ -442,26 +437,14 @@ export const ConsolePlugin = ({
 
   useEffect(() => {
     const cleanups = [
-      registerComponent(
-        ConsolePanel.COMPONENT,
-        (ConsolePanel as unknown) as ComponentType
-      ),
-      registerComponent(
-        CommandHistoryPanel.COMPONENT,
-        (CommandHistoryPanel as unknown) as ComponentType
-      ),
+      registerComponent(ConsolePanel.COMPONENT, ConsolePanel),
+      registerComponent(CommandHistoryPanel.COMPONENT, CommandHistoryPanel),
       registerComponent(
         FileExplorerPanel.COMPONENT,
-        (FileExplorerPanel as unknown) as ComponentType
-      ),
-      registerComponent(
-        LogPanel.COMPONENT,
-        (LogPanel as unknown) as ComponentType
-      ),
-      registerComponent(
-        NotebookPanel.COMPONENT,
-        (NotebookPanel as unknown) as ComponentType
+        (FileExplorerPanel as unknown) as PanelComponentType
```

Comment:
@mattrunyon any idea how to get TypeScript typings correct so we don't need to explicitly convert this one? I'm having a hard time getting TypeScript to co-operate here.

Reply:
The error:
```
Argument of type 'ConnectedComponent<typeof FileExplorerPanel, Omit<ClassAttributes<FileExplorerPanel> & PanelProps & { localDashboardId: string; metadata?: Record<...> | undefined; } & StateProps & DispatchProp<...>, "dispatch" | ... 2 more ... | "fileStorage"> & PanelProps & { ...; }>' is not assignable to parameter of type 'PanelComponentType<ComponentClass<{}, any>, Omit<ClassAttributes<FileExplorerPanel> & PanelProps & { localDashboardId: string; metadata?: Record<...> | undefined; } & StateProps & DispatchProp<...>, "dispatch" | ... 2 more ... | "fileStorage"> & PanelProps & { ...; }>'.
  Type 'ConnectedComponent<typeof FileExplorerPanel, Omit<ClassAttributes<FileExplorerPanel> & PanelProps & { localDashboardId: string; metadata?: Record<...> | undefined; } & StateProps & DispatchProp<...>, "dispatch" | ... 2 more ... | "fileStorage"> & PanelProps & { ...; }>' is not assignable to type 'FunctionComponent<{}>'.
    Types of parameters 'props' and 'props' are incompatible.
      Type '{ children?: ReactNode; }' is not assignable to type 'Omit<ClassAttributes<FileExplorerPanel> & PanelProps & { localDashboardId: string; metadata?: Record<string, unknown> | undefined; } & StateProps & DispatchProp<...>, "dispatch" | ... 2 more ... | "fileStorage"> & PanelProps & { ...; }'.
        Type '{ children?: ReactNode; }' is missing the following properties from type 'Omit<ClassAttributes<FileExplorerPanel> & PanelProps & { localDashboardId: string; metadata?: Record<string, unknown> | undefined; } & StateProps & DispatchProp<...>, "dispatch" | ... 2 more ... | "fileStorage">': glContainer, glEventHub, localDashboardIdts(2345)
```